modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -53,6 +53,8 @@ const float TILE_SIZE = 512.0;
 const float PI = 3.1415926536;
 const float WORLD_SCALE = TILE_SIZE / (PI * 2.0);
 const vec3 ZERO_64_LOW = vec3(0.0);
+const float EARTH_RADIUS = 6370972.0;
+const float GLOBE_RADIUS = 256.0;
```

Comment:
Add comments about units etc. Especially globe radius is not automatically clear. Why do we want globes to be in 256 units, some relation to flat WebMercator probably and maybe some choices we make in common space?

Reply:
1 common unit maps to 1 screen pixel at zoom=0. It is indeed arbitrary but offers roughly the same "zoom level" expectation as the web mercator projection.